docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -74,14 +74,51 @@ bigquery.project-id=<your Google Cloud Platform project id>
 
 ### Multiple GCP projects
 
-The BigQuery connector can only access a single GCP project.Thus, if you have
+The BigQuery connector can only access a single GCP project. Thus, if you have
 data in multiple GCP projects, You need to create several catalogs, each
 pointing to a different GCP project. For example, if you have two GCP projects,
 one for the sales and one for analytics, you can create two properties files in
 `etc/catalog` named `sales.properties` and `analytics.properties`, both
 having `connector.name=bigquery` but with different `project-id`. This will
 create the two catalogs, `sales` and `analytics` respectively.
 
+(bigquery-project-id-resolution)=
+### Project ID resolution
+
+The BigQuery connector can be configured to use different project for billing, 
+see more in the 
+[Google Cloud documentation](https://cloud.google.com/iam/docs/attach-service-accounts).
```

Comment:
why that link .. maybe we need to change the link title to mention service accounts..

Reply:
Removed this sentence per https://github.com/trinodb/trino/pull/25109#discussion_r1970331942